src/main/java/com/daml/extensions/testing/utils/SandboxUtils.java

```diff
@@ -17,11 +19,10 @@
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
 import java.util.concurrent.atomic.AtomicInteger;
-import org.slf4j.Logger;
 
 public class SandboxUtils {
   private static final Path DAML_YAML = Paths.get("daml.yaml");
-  private static Range<Integer> SANDBOX_PORT_RANGE = Range.closed(6860, 6890);
+  private static Range<Integer> SANDBOX_PORT_RANGE = Range.closed(6860, 6864);
```

Comment:
Is it possible to have a somewhat bigger port range?

Reply:
Connection problems come from 6866-6868 ports. I can keep wide range as before but exclude this "broken" ports from the range

Reply:
that's also good. why are 6866-6868 ports special?